contracts/libs/RollupUtils.sol

```diff
@@ -501,6 +531,19 @@ library RollupUtils {
         return abi.decode(txBytes, (uint256, uint256, uint256, bytes));
     }
 
+    function DecompressTransfers(bytes memory txs)
```

Comment:
I added the decompress util for transfer type only. This is because we want the fraud proof test to pass. Will add other utils for other tx types

Reply:
We can replace the function body with the API from #111 too